components/profile_popover/profile_popover.jsx

```diff
@@ -311,17 +304,24 @@ class ProfilePopover extends React.Component {
         const email = this.props.user.email;
         if (email) {
             dataContent.push(
-                <div
-                    data-toggle='tooltip'
-                    title={email}
-                    key='user-popover-email'
-                >
-                    <a
-                        href={'mailto:' + email}
-                        className='text-nowrap text-lowercase user-popover__email padding-bottom half'
+                <div>
```

Comment:
This needs a key or else it'll cause a JavaScript warning because it's part of the `dataContent` array. Similarly, you can remove the keys from the hr and tooltip div below since they aren't directly in the array any more.
```javascript
if (email) {
    dataContent.push(
        <div key=...>
              ...
        </div>
    );
}
```

Alternatively, you could just add them into the array on their own to get rid of the extra div wrapping them
```javascript
if (email) {
    dataContent.push(
        <hr key='user-popover-hr'/>
    );

    dataContent.push(
        <div key='user-popover-email' />
    )
}
```